src/common/loki.h

```diff
@@ -37,6 +37,7 @@ namespace loki
 double      round           (double);
 double      exp2            (double);
 std::string hex64_to_base32z(std::string const& src);
+uint64_t    clamp_u64       (uint64_t min, uint64_t val, uint64_t max);
```

Comment:
The order here is bound to be confused at some point because `std::clamp` (which does the same thing, but needs C++17) takes arguments as `(val, min, max)`.